entrypoint.sh

```diff
@@ -5,30 +5,34 @@ export ALLOWLIST_DIR=/allowlists
 export PYPI_ALLOWLIST="$ALLOWLIST_DIR"/pypi.allowlist
 export CRAN_ALLOWLIST="$ALLOWLIST_DIR"/cran.allowlist
 
+timestamp() {
+    date -Is
+}
+
 # Ensure allowlist files exist
 if ! [ -f "$PYPI_ALLOWLIST" ]; then
-    echo "PyPI allowlist not found"
+    echo "$(timestamp) PyPI allowlist not found"
```

Comment:
Is this simpler than just having `$(date -Is)` here?

Reply:
Avoids duplicating the same `date` command throughout the script.